### PR TITLE
Amsterdam file with some attractions

### PR DESCRIPTION
**Pull request by Gjambhulkar**
This is the beginning of the Amsterdam file :1234: 

Please provide feedback on:
- Don't suggest the spelling mistakes, I know it's there :smile:

### CODE REVIEW

File:  Gjambhulkar-Amsterdam.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ # Amsterdam, NL
+
+ ### Restaurants
+
+ ### Local Attraction
+
+ - Red Light District :)
+ - Sex Musium :))
```
Comment (Gjambhulkar) [inline  Gjambhulkar-Amsterdam.md]: it should be Museum

File:  Gjambhulkar-Amsterdam.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ # Amsterdam, NL
+
+ ### Restaurants
+
+ ### Local Attraction
```
Comment (pdeleon-ctg) [inline  Gjambhulkar-Amsterdam.md]: Local attractions sound fun! 😆

Comment (Gjambhulkar) [inline  Gjambhulkar-Amsterdam.md]: :smile:

**Comment by pdeleon-ctg** [comment]
👍 :shipit:

**Pull request merged**